main.js

```diff
@@ -56,7 +56,34 @@ const VideoList = {
           source:'https://www.youtube.com/embed/MhkGQAoc7bc',
           author: 'Someone\'s Buddy'  
         }
-      ]
+      ],
+      navbarItems: [
+        {
+          name: 'Home',
+          src: '/',
+        },
+        {
+          name: 'Welcome',
+          src: 'https://welcome.realdevsquad.com/',
+        },
+        {
+          name: 'Events',
+          src: './events.html',
```

Comment:
will this be hosted on the learning site?

Reply:
This is navbarItems data which is used by navbar-item component to generate item of navbar.

Reply:
I think he's referring to it being linked relatively to the current site(learn)
Which is causing the link to break since it should be redirecting to `https://www.realdevsquad.com/events.html` 

Could you fix that, please?

Reply:
I think he's referring to it being linked relatively to the current site(learn)
Which is causing the link to break since it should be redirecting to `https://www.realdevsquad.com/events.html` 

Could you fix that, please?

Reply:
Fixed!